irods_specific_implementation/irods_rest_query_api_implementation.h

```diff
@@ -39,6 +39,12 @@ namespace irods::rest
                 auto conn = get_connection(_request.headers().getRaw("authorization").value(), _query_string);
 
                 std::string query_string{decode_url(_query_string)};
+                if ("0" == _case_sensitive) {
+                    std::transform(query_string.begin(),
+                                    query_string.end(),
+                                    query_string.begin(),
+                                    ::toupper);
```

Comment:
This needs a small tweak. Please see the following: https://en.cppreference.com/w/cpp/string/byte/toupper